### ISSUE TITLE
Is BN running in eval mode during test time?

### ISSUE THREAD

**Issue by mikolchon** [DmitryUlyanov/texture_nets#86]
Hi and thanks for the great work. I am not familiar with lua torch so I better ask: is the BN layer in "train" or "eval" mode during testing? I know that in GANs they keep the BN layer in train mode always for stability https://discuss.pytorch.org/t/why-dont-we-put-models-in-train-or-eval-modes-in-dcgan-example/7422/2
How is it implemented in your code?

**Comment by DmitryUlyanov** [comment]
Initially we were using `train` mode for BN at test time and observed a similar effect of an `eval` mode as in GAN's. We then proposed to use Instance Normalization [1] instead of BatchNorm, which works in the same way for train and test. So this repo uses Instance Norm, not BN. 

[1] https://arxiv.org/abs/1701.02096
